README.md

```diff
@@ -129,15 +131,16 @@ The HTTPX project relies on these excellent libraries:
   * `h11` - HTTP/1.1 support.
 * `certifi` - SSL certificates.
 * `idna` - Internationalized domain name support.
-* `sniffio` - Async library autodetection.
 
 As well as these optional installs:
 
-* `h2` - HTTP/2 support. *(Optional, with `httpx[http2]`)*
-* `socksio` - SOCKS proxy support. *(Optional, with `httpx[socks]`)*
-* `rich` - Rich terminal support. *(Optional, with `httpx[cli]`)*
-* `click` - Command line client support. *(Optional, with `httpx[cli]`)*
-* `brotli` or `brotlicffi` - Decoding for "brotli" compressed responses. *(Optional, with `httpx[brotli]`)*
+* `anyio`, `sniffio` - Async support for `asyncio`. *(Optional, with `httpx['asyncio']`)*
+* `trio`, `sniffio` - Async support for `trio`. *(Optional, with `httpx['trio']`)*
```

Comment:
```suggestion
* `anyio` - Async support for `asyncio`. *(Optional, with `httpx['asyncio']`)*
* `trio` - Async support for `trio`. *(Optional, with `httpx['trio']`)*
```

It's not in `pyproject.toml` and httpcore also doesn't mention it in readme. Actually, it's dependency of dependencies and if you want to mention it here, `trio` also have other dependencies can be here.